openapi/paths/credit-memos@{id}.yaml

```
@@ -69,3 +69,35 @@ put:
       $ref: ../components/responses/NotFound.yaml
     '422':
       $ref: ../components/responses/ValidationError.yaml
+patch:
+  x-products:
+    - Core
+  tags:
+    - Credit memos
+  summary: Patch a credit memo and/or allocations with specified ID
```

nitpick: It’s best to avoid using the same verb as the HTTP request in the summary and descriptions (based on our style)
```suggestion
  summary: Update credit memo, allocation, or both
```